openbas-front/src/admin/components/settings/data_ingestion/ImportUploaderMapper.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useAppDispatch } from '../../../../utils/hooks';
+import ImportUploader from '../../../../components/common/ImportUploader';
+import { importMapper } from '../../../../actions/mapper/mapper-actions';
+
+const ImportUploaderMapper = () => {
+  // Standard hooks
+  const dispatch = useAppDispatch();
```

Comment:
We don't have mapper in store right ?
So we don't need to dispatch it

Reply:
Good point, I based this on ImportUploaderExercise which does have one. I'm removing it

Reply:
Done :)